File: Crio/crio_programming_interview_problems-master/UniqueString/UniqueString.py
def is_unique(s):
    res = False
# CRIO_SOLUTION_START_MODULE_L1_PROBLEMS
    lst = [x[1] for x in list(enumerate(s))]
    res =  len(s) == len(set(lst))
# CRIO_SOLUTION_END_MODULE_L1_PROBLEMS
    return res
def main():
    s = input().strip()

    if is_unique(s):
        print("Yes")
    else:
        print("No")


if __name__ == '__main__':
    main()
# CRIO_SOLUTION_START_MODULE_L1_PROBLEMS
    assert(not is_unique('a' * 450000 + 'b' * 72300))
# CRIO_SOLUTION_END_MODULE_L1_PROBLEMS

File: Crio/crio_programming_interview_problems-master/UniqueString/UniqueString.cpp
#include<bits/stdc++.h>
using namespace std;


bool isUnique(string s) {
// CRIO_SOLUTION_START_MODULE_L1_PROBLEMS
	int count[300];
	memset(count , 0 , sizeof count);
	for(int i = 0 ; i < s.size() ; i++) {
		if(count[s[i]] >= 1) {
			return false;
		}
		count[s[i]]++;
	}
// CRIO_SOLUTION_END_MODULE_L1_PROBLEMS
	return true;
}

int main() {
	string s;
	cin >> s;
	bool answer = isUnique(s);
	if(answer) {
		cout << "Yes";
	} else {
		cout << "No";
	}
	return 0;
}
